src/ORM/README.md

```diff
@@ -35,7 +35,7 @@ mappers if no explicit connection is defined.
 ## Creating Associations
 
 In your table classes you can define the relations between your tables. CakePHP
-supports 4 association types out of the box:
+ORM supports 4 association types out of the box:
```

Comment:
"The CakePHP ORM" is okay as-is. It's also okay as-is for the header. Only this occurrence needs cahnged it since it's being used in a possessive manner.